fury/tests/test_ui.py

```diff
@@ -1145,6 +1145,17 @@ def _on_change():
         show_manager.scene.add(filemenu)
         show_manager.start()
 
+def test_ui_combobox_2d(interactive=False):
+    values = ["An Item" for i in range(0,5)]
```

Comment:
`["An Item" + str(i) for i in range(0,5)]` instead. Your item will be more visible

Reply:
I though the scrolling was not working xD